qcengine/programs/gamess/harvester.py

```diff
@@ -445,6 +537,10 @@ def harvest_outfile_pass(outtext):
         qcvar["CURRENT REFERENCE ENERGY"] = qcvar["DFT TOTAL ENERGY"]
         qcvar["CURRENT ENERGY"] = qcvar["DFT TOTAL ENERGY"]
 
+    if "EFP TOTAL ENERGY" in qcvar:
+        qcvar["CURRENT REFERENCE ENERGY"] = qcvar["EFP TOTAL ENERGY"]
```

Comment:
unless something needs it, maybe drop the current ref line. it's questionable.

Reply:
I had to add that line since `CURRENT REFERENCE ENERGY` is expected here:
https://github.com/MolSSI/QCEngine/blob/4525eb3ea94557c7f9aed400a02bf98a95ce5c92/qcengine/programs/gamess/runner.py#L155

What exactly is questionable about it? Would setting `CURRENT ENERGY` make more sense? Does `CURRENT REFERENCE ENERGY` imply a full QM calculation?

Reply:
If you could get away with `CURRENT ENERGY` only that'd be better, but if `CURRENT REFERENCE ENERGY` needed, fine. It's questionable b/c it's an interaction energy like SAPT rather than a total energy.